bson-adt-core/src/test/scala/adt/bson/scalacheck/CommonGenerators.scala

```diff
@@ -55,11 +32,4 @@ trait CommonGenerators {
       date <- arbDate.arbitrary
     } yield new ObjectId(date)
   }
-
-  // The only way in which the ObjectId could cause a property to fail is if someone is using
-  // the timestamp from the ObjectId in a test. So we only need to shrink the timestamp.
-  implicit val shrinkObjectId: Shrink[ObjectId] = Shrink { oid =>
-    val shrinkSeconds = shrinkSameSign(oid.getTimestamp)
-    shrinkSeconds map { millis => new ObjectId(new java.util.Date(millis * 1000)) }
-  }
```

Comment:
I recant my statement in the comment in that I don't think there is a good way to shrink `ObjectId`. The default of not shrinking is likely to have better results in most cases.